tox.ini

```diff
@@ -30,7 +30,7 @@ deps =
 # The following indicates which extras_require from setup.cfg will be installed
 extras =
     test
-
+    all
```

Comment:
Why have you added this?

Reply:
I was facing a local venv issue, sorry I'll fix this